prov/sm2/src/sm2_progress.c

```diff
@@ -91,18 +92,18 @@ static int sm2_start_common(struct sm2_ep *ep,
 	}
 
 	comp_buf = rx_entry->iov[0].iov_base;
-	comp_flags = sm2_rx_cq_flags(xfer_entry->op, rx_entry->flags,
-				     xfer_entry->op_flags);
+	comp_flags = sm2_rx_cq_flags(xfer_entry->hdr.op, rx_entry->flags,
+				     xfer_entry->hdr.op_flags);
 
 	if (err) {
 		FI_WARN(&sm2_prov, FI_LOG_EP_CTRL, "Error processing op\n");
 		ret = sm2_write_err_comp(ep->util_ep.rx_cq, rx_entry->context,
 					 comp_flags, rx_entry->tag, err);
 	} else {
-		ret = sm2_complete_rx(ep, rx_entry->context, xfer_entry->op,
-				      comp_flags, total_len, comp_buf,
-				      xfer_entry->sender_gid, xfer_entry->tag,
-				      xfer_entry->cq_data);
+		ret = sm2_complete_rx(
+			ep, rx_entry->context, xfer_entry->hdr.op, comp_flags,
```

Comment:
ugh why is the formatting like this?

Reply:
We can play with clang format more in the future... nothing we can do about this on this commit.